src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -130,6 +130,7 @@ struct RawTexMetadata {
     uint16_t width, height;
     float h_byte_scale = 1, v_pixel_scale = 1;
     std::shared_ptr<Ship::Texture> resource;
+    std::string name;
```

Comment:
This reintroduces the removed texture name field. I reintroduced this because the replacement, `std::shared_ptr<Ship::Texture> resource`, does not have good way of testing equivalence. Games will be specifying textures to be replaced via file paths, and there is no good way of going from paths -> resources is the first place, so back to paths it is.

Reply:
Is name the same as path? If it's the same then the resource has the path in it's InitData.

Reply:
Using that did work. Strangely there were some textures that did not have an associated resource, but a null check fixed that.

Reply:
Sounds good to me.

Reply:
Any incite into the cases where the Texture did not have a resource? I assume this isn't the Texture subclass of Resource.

Reply:
Roze on Discord:
```
I'm not sure, it also did not have a name. It occurred during boot before anything was drawn. It may have not even been a texture, just a load_block before anything was set up? I didn't look into it too much.
```